docker-compose.yml

```diff
@@ -74,7 +82,7 @@ services:
     volumes:
     - solr-data:/var/solr
     ports:
-    - ${SOLR_PORT:-8983}:8983
+    - 8983:8983
```

Comment:
@whereismyjetpack don't we want to revert this change so we can change the port easily?

Reply:
doh!